.vitest/src/instances.ts

```diff
@@ -109,14 +109,17 @@ function defineInstance(params: DefineInstanceParams) {
 
   const bundlerServer = createServer({
     instance: (key) =>
-      rundler({
-        binary: rundlerBinaryPath,
-        entryPointVersion: "0.6.0",
-        nodeHttp: `http://127.0.0.1:${anvilPort}/${key}`,
-        rpc: {
-          api: "eth,rundler,debug",
+      rundler(
+        {
+          binary: rundlerBinaryPath,
+          entryPointVersion: "0.6.0",
```

Comment:
Should we be defaulting this to 0.7.0.?

Reply:
the rundler defaults to 0.7.0 in prool, but adding this temporarily. Once I get: https://github.com/wevm/prool/pull/26 in we won't even need 2 instances for the rundler and will be able to have the rundler run both entrypoints

Reply:
was mainly just being explicit here, I add another instance for 070 here: https://github.com/alchemyplatform/aa-sdk/pull/866